pkg/dns/azure/client/auth.go

```diff
@@ -2,15 +2,17 @@ package client
 
 import (
 	"errors"
-	"os"
-	"strings"
-
+	"fmt"
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure"
+	"os"
+	"strings"
```

Comment:
Mind bumping these back up to the original grouping?

Reply:
Yes, sorry, I'm not sure how those got moved around (assuming my IDE did it).